admin/partials/acc_user_importer-admin-settings.php

```diff
@@ -87,6 +87,14 @@ function accUM_get_ex_user_role_value_default()
 {
     return "subscriber";
 }
+function accUM_get_when_2_delete_ex_user_default()
+{
+    return 9999;
```

Comment:
Maybe add another 9?  30 years is a long time... until it isn't.

Reply:
I was trying to use "9999" days as an indirect way to turn off the feature. Now that I saw your request to plain turn off the feature, I agree. It is nicer and safer to have simple setting to enable/disable the feature. This way if there is a bug in the logic, we can disable and it will just skip the whole thing.  On the next commit, I will set the the default to feature=disabled, and I will set the "when to delete ex-user" default value to something more sensible, like 1 year.  So installing the new version will have no change in behavior compared to before, and if the new feature is enabled, the time before deletion will by default be 1 year.

Reply:
Done, I just uploaded the new code to address your review.